src/targets/gpu/abs.cpp

```diff
@@ -7,7 +7,7 @@ namespace gpu {
 
 shape miopen_abs::compute_shape(const std::vector<shape>& inputs) const
 {
-    check_shapes{inputs, *this}.has(2).not_broadcasted();
+    check_shapes{inputs, *this}.has(2).packed();
```

Comment:
I dont think miopen can handle transposed shapes, but it should handle non-packed shapes. It should probably be changed to `.not_broadcasted().not_transposed()`.

Reply:
I just tried, looks like the MIOpen can handle the transposed shape, so I will keep the current implementation.